receiver/mongodbatlasreceiver/mongodb_event_to_logdata.go

```diff
@@ -0,0 +1,167 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mongodbatlasreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mongodbatlasreceiver"
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mongodbatlasreceiver/internal/model"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.uber.org/zap"
+)
+
+const (
+	// Number of log attributes to add to the plog.LogRecordSlice.
+	totalLogAttributes = 5
+
+	// Number of resource attributes to add to the plog.ResourceLogs.
+	totalResourceAttributes = 4
+)
+
+const layout = "2006-01-02T15:04:05.000-07:00"
```

Comment:
Maybe a comment here just to call it out that this is the timestamp format in the logs.